src/pages/Explore.jsx

```diff
@@ -1,8 +1,30 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 import Header from '../components/Header';
 
 function Explore() {
-  return <Header title="Explorar" />;
+  return (
+    <div>
```

Comment:
```suggestion
    <>
```
Acho interessante utilizarmos fragments para diminuir o numero de divs no código, o que acha?